src/main/java/calculator/StringAndCalculator.java

```diff
@@ -0,0 +1,47 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class StringAndCalculator {
+    private final static String ABNORMAL_MATCH_PATTERN = "//(.)\n(.*)";
+    private final static String NORMAL_MATCH_PATTERN = "[,:]";
+
+    public static int splitAndSum(String text) {
+        if (Objects.isNull(text) || text.equals("")) {
+            return 0;
+        }
+
+        return sumIntegers(stringsToIntegers(textToStrings(text)));
+    }
+
+    private static String[] textToStrings(String text) {
+        Matcher m = Pattern.compile(ABNORMAL_MATCH_PATTERN).matcher(text);
+        if (m.find()) {
+            String customDelimiter = m.group(1);
+            return m.group(2).split(customDelimiter);
+        } else {
+            return text.split(NORMAL_MATCH_PATTERN);
+        }
+    }
+
+    private static List<Integer> stringsToIntegers(String[] values) {
+        return Arrays.stream(values).map(StringAndCalculator::stringToInt).collect(Collectors.toList());
```

Comment:
`Stream`을 사용한다면 가독성을 위해 `.`을 한 줄에 하나씩 찍어보는 건 어떨까요?
```suggestion
        return Arrays.stream(values)
            .map(StringAndCalculator::stringToInt)
            .collect(Collectors.toList());
```